api/app/postgresql.js

```diff
@@ -42,14 +56,14 @@ const importData = () => {
       });
       await client.query("COMMIT");
     } catch (e) {
-      console.log("Error");
+      logger.error('search error: ', err);
       await client.query("ROLLBACK");
       throw e;
     } finally {
-      console.log("Release");
+      logger.error("Release");
```

Comment:
why is this an error? 

```suggestion
      logger.info("Release");
```